DRAFT_RELEASE_NOTES.md

```diff
@@ -1,5 +1,8 @@
 # Major Additions
 
+## Helm Migration
+The Spark Operator Helm chart has been migrated to the `v2` structure.  To migrate your helm charts to use the v2 pattern, please follow the [instruction](https://boozallen.github.io/aissemble/aissemble/current/containers.html#_kubernetes_artifacts_upgrade) in the technical documentation.
```

Comment:
A: IMO, instead of approaching this like we would a normal v2 migration, we should say the issue we're patching (that the v1 chart is not compatible with 1.8) and clarify that projects created after 1.2.10 will already have the v2 chart.